src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -4027,6 +4027,8 @@ private static bool IsParameterModifier(SyntaxKind kind)
 
         private void ParseParameterModifiers(SyntaxListBuilder modifiers)
         {
+            var seenRefModifier = false;
```

Comment:
I see we use both styles in the codebase. I usually prefer `var` unless the explicit type improves code readability.